server/hl7/hl7.controller.js

```diff
@@ -0,0 +1,36 @@
+const multer = require('multer');
+const HL7 = require('./hl7.model');
+const APIError = require('../helpers/APIError');
+const httpStatus = require('http-status');
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => { cb(null, 'data/hl7-uploads'); },
+  filename: (req, file, cb) => (cb(null, `${file.originalname}-${Date.now()}`))
```

Comment:
this adds the date after the file extension, i.e. `originalfilename.txt-123456789`, not a big deal, but we probably want the file extension to stay at the end of the name

Reply:
Yes, this is intentional for now. Just wanted to create something for other to work from. Changes to this will be coming in another PR.